tests/test2/test2.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jun 30 11:11:25 2021
+
+@author: pearsonra
+"""
+
+import unittest
+import json
+import pathlib
+import shapely
+import geopandas
+import shutil
+import numpy
+
+from src.geofabrics import lidar_fetch
+from src.geofabrics import geometry
+
+class Test2(unittest.TestCase):
```

Comment:
Might be worth considering a test class naming convention in the future, so that it's easier to find where to slot new unit tests in

Reply:
I was wondering about that.
Could go with a folder hierarchy
integraton
-> test 1
lidar_fetch
-> test 1

or with names that are more descriptive. Are there any packages that you think do a good job with their conventions?

Reply:
Anyway - I think you're right both about a better naming convention so it's clearer where to go to find/put a particular test, and also about in the future. I'll defer this until be have a few more tests and then hopefully it will be clearer how to organise the tests :)

Reply:
I don't have any in mind right now, although I was thinking of something like this
An equivalent structure would be
/lidar_fetch/
->`LidarFetchTest`
--->`test_num_files_downloaded()`
--->`test_size_of_files_correct()`
--->`...`

Or inspired by Leapfrog block tests
/lidar_fetch/
->`LidarFetchTest`
--->`lidar_fetch100()`
--->`lidar_fetch200()`
--->`...`

Reply:
Thanks for this and your comment below. I've updated the test2 -> lidar_fetch/OpenTopographyTest and split out the relevent tests as suggested. It looks much better now, thanks!